node/src/database/db_initializer.rs

```diff
@@ -1032,6 +1039,7 @@ mod tests {
             false,
         );
         verify(&mut config_vec, "mapping_protocol", None, false);
+        verify(&mut config_vec, "min_hops_count", Some("3"), false);
```

Comment:
Last week I created this Card for the same. I did that because it could be a good first issue.

https://github.com/MASQ-Project/MASQ-Node-issues/issues/715

Feel free to edit it.

Reply:
Okay. I did.